external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -542,14 +543,26 @@ def load(cls, path: str) -> "PureKerasModel":
 
     def predict(self, X: xr.Dataset) -> xr.Dataset:
         """Predict an output xarray dataset from an input xarray dataset."""
-        inputs = [X[name].values for name in self.input_variables]
+        X_tmp = _rename_sample_dim(X, self.sample_dim_name)
```

Comment:
This line uses `self.sample_dim_name` but some other lines use `SAMPLE_DIM_NAME`. It doesn't matter too much which one you use, but please change them to be consistent. If you use `SAMPLE_DIM_NAME`, please remove `sample_dim_name` from the `__init__`.